test/cmake_install_test/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright 2018 Peter Dimov
+# Distributed under the Boost Software License, Version 1.0.
+# See accompanying file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt
+
+cmake_minimum_required(VERSION 3.5...3.16)
```

Comment:
Top-level CMakeLists.txt requires cmake 3.8, probably makes sense to require the same here too.